ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml

```diff
@@ -23,22 +23,22 @@ base_images:
     name: test
     namespace: ocp-kni
     tag: dev-scripts
-  openshift_release_golang-1.16:
+  openshift_release_golang-1.17:
```

Comment:
``cloud_hotfix_releases`` is still pointing to ``2.1.x``, so maybe we want to remain with the same golang version in this file